TEditXna/View/WorldRenderXna.xaml.cs

```diff
@@ -176,21 +176,21 @@ private void xnaViewport_LoadContent(object sender, GraphicsDeviceEventArgs e)
 
         private void LoadResourceTextures(GraphicsDeviceEventArgs e)
         {
-            _textures.Add("Spawn", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.spawn_marker.png", e.GraphicsDevice));
-            _textures.Add("Dungeon", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.dungeon_marker.png", e.GraphicsDevice));
-            _textures.Add("Old Man", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_old_man.png", e.GraphicsDevice));
-            _textures.Add("Arms Dealer", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_arms_dealer.png", e.GraphicsDevice));
-            _textures.Add("Clothier", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_clothier.png", e.GraphicsDevice));
-            _textures.Add("Demolitionist", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_demolitionist.png", e.GraphicsDevice));
-            _textures.Add("Dryad", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_dryad.png", e.GraphicsDevice));
-            _textures.Add("Guide", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_guide.png", e.GraphicsDevice));
-            _textures.Add("Merchant", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_merchant.png", e.GraphicsDevice));
-            _textures.Add("Nurse", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_nurse.png", e.GraphicsDevice));
+            _textures.Add("Spawn"          , WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.spawn_marker.png", e.GraphicsDevice));

```

Comment:
This would be my second choice:

        _textures.Add("Spawn",           WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.spawn_marker.png",      e.GraphicsDevice));
        _textures.Add("Dungeon",         WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.dungeon_marker.png",    e.GraphicsDevice));
        _textures.Add("Old Man",         WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_old_man.png",       e.GraphicsDevice));
        _textures.Add("Arms Dealer",     WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_arms_dealer.png",   e.GraphicsDevice));
        _textures.Add("Clothier",        WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_clothier.png",      e.GraphicsDevice));
        _textures.Add("Demolitionist",   WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_demolitionist.png", e.GraphicsDevice));
        _textures.Add("Dryad",           WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_dryad.png",         e.GraphicsDevice));
        _textures.Add("Guide",           WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_guide.png",         e.GraphicsDevice));
        _textures.Add("Merchant",        WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_merchant.png",      e.GraphicsDevice));
        _textures.Add("Nurse",           WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_nurse.png",         e.GraphicsDevice));
        _textures.Add("Goblin Tinkerer", WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_goblin.png",        e.GraphicsDevice));
        _textures.Add("Wizard",          WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_wizard.png",        e.GraphicsDevice));
        _textures.Add("Mechanic",        WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_mechanic.png",      e.GraphicsDevice));
        _textures.Add("Santa Claus",     WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.npc_santa_claus.png",   e.GraphicsDevice));
        _textures.Add("Grid",            WriteableBitmapEx.ResourceToTexture2D("TEditXna.Images.Overlays.grid.png",              e.GraphicsDevice));